src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReaderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+use Symfony\Component\Filesystem\Path;
+
+interface ImportMapConfigReaderInterface
+{
+    /**
+     * @return ImportMapEntries<ImportMapEntry>
```

Comment:
`ImportMapEntries` already defines `@template-implements \IteratorAggregate<ImportMapEntry>`, which is why you get that Psalm error. You may just remove these `@return` and static analysis tools should still work :+1: